src/app/dashboard/dashboard.component.scss

```diff
@@ -0,0 +1,81 @@
+@import "../../global_variables";
+
+// kashti sidebar styles
+#sidebar {
+  background: #37b0e7 url('../../assets/images/topo-pattern-white.png') repeat 20px 0;
+  background-size: 376px auto;
+  width: 56px;
```

Comment:
Would like to convert px statements like this to `em` or %; in this case we'd also want to center the logo and icons. Thoughts?

Reply:
Fine with me. It's based on the logo width, a one-off measurement in the UI.